src/Profile.js

```diff
@@ -141,6 +141,14 @@ class Profile {
     }, 'application/json', 'json');
   }
 
+  getGoals() {
```

Comment:
Keeping the naming convention for lists as above, can this just be `goals()` please?